spec/factories/incidental_type.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :incidental_type do
+    sequence(:name) { |n| "name #{n}" }
```

Comment:
Actually now that you bring it up a uniqueness validation on a incidental type is probably a good idea, unless we want multiple types of the same name with different fees